STEERING-COMMITTEE.md

```diff
@@ -162,13 +117,157 @@ There are two ways to raise issues to the steering committee for decision:
+Six weeks or more before the election, the Steering Committee shall appoint
+Election Officer(s) (see below).  Four weeks or more before the election, the
+Election Officer(s) will issue a call for nominations, publish the list of
+voters, and open the call for exceptions.  One week before the election the call
+for nominations and exceptions will be closed.  The election will be open for
+voting not less than two weeks and not more than four.  The results of the
+election shall be announced within one week of closing the election.  New
+Steering Committee members will take office on December 1 of each year.
+
+### Election Officer(s)
+
+Six weeks or more before the election, the Steering Committee will appoint
+between one and three Election Officer(s) to administer the election.  Elections
+Officers will be Knative community members in good standing who are eligible to
+vote, are not running for Steering in that election, and can make a public
+promise of impartiality.  They will be responsible for:
+
+* Making all announcements associated with the election
+* Preparing and distributing electronic ballots
+* Judging exception requests
+* Assisting candidates in preparing and sharing statements
+* Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote
+
+Anyone who has at least 50 contributions in the last 12 months is eligible to
+vote in the Steering election. Contributions are defined as opening PRs,
+reviewing and commenting on PRs, opening and commenting on issues, writing
+design docs, commenting on design docs, helping people on slack, participating
+in working groups, and other efforts that help advance Knative the project.
+
+[This
+dashboard]((https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year)
+shows only GitHub based contributions and does not capture all the contributions
+we value. We expect this metric not to capture everyone who should be eligible
+to vote. If a community member has had significant contributions over the past
+year but is not captured in the [stats.knative.dev
+dashboard](https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year),
+they will be able to submit an exception form to the Elections Officer(s) who
+will then review and determine whether this member should be marked as an
+exception.  All exceptions, and the reasons for them, will be recorded in a log
+that will be available to Steering and the TOC.
+
+All eligible voters will be captured at
+knative/community/steering-elections/$YEAR/voters.md and the voters’ guide will
+be captured at knative/community/steering-elections/$YEAR/README.md, similar to
+the kubernetes election process and identical to the TOC election process.
+
+We are committed to an inclusive process and will adapt future eligibility
+requirements based on community feedback.
+
+### Candidate Eligibility
```

Comment:
Im not sure if I missed it (sorry if I did :)), but I can't see here whether we've specified whether someone can be on both the TOC and the Steering committee concurrently? Can a TOC member run? If they can and they win are the expected to step down? etc

Reply:
Good question!  I've wondered the same before and am mixed on this, but curious what prior art there is, since I could make a case for either. 😅

Reply:
yah, I feel like it'd be pretty odd to have to think "hmm better not vote for Matt for TOC since then he can't be on Steering" ;). On the other hand, allowing people to be on both committees seems to make having two committees pretty weird (+ if Steering is supposed to be an escalation point for TOC, which it currently is in this doc, how does that even work when many members are the same?). 

I personally lean towards letting people who want to be on both be on both, but either way, I feel like we should be explicit about this, so very interested in opinions and precedents also!

Reply:
I'm also assuming that a candidate could step down at any moment, or conditionally resign from one committee in order to serve on the other if elected.

I do think serving on both committees might be a bit much, but I'm not sure I'd want to explicitly prohibit it.

Reply:
I don't think it should be prohibited. In the event someone is qualified, wants to do both roles, and the community wants it, why not?

Reply:
I'm making it explicit that it's okay. Thanks for raising this @julz